default/stringtables/ru.txt

```diff
@@ -507,7 +508,7 @@ SD_BASE_DECOY
 Спутник связи
 
 SD_BASE_DECOY_DESC
-Невооруженный спутник, который может активировать планетарную защиту даже против пассивных врагов, однако, при ее отсутствии ничем не примечателен и может быть уничтожен на пути вражеского огня. Может быть построен на любой колонизированной планете для чего не требуется [[buildingtype BLD_SHIPYARD_BASE]]
+Невооруженный спутник, который может активировать планетарную защиту даже против пассивных врагов, однако, при её отсутствии, ничем не примечателен и может быть уничтожен на пути вражеского огня. Может быть построен на любой колонизированной планете, для чего не требуется [[buildingtype BLD_SHIPYARD_BASE]].
```

Comment:
Здесь была попытка перевести как можно ближе к оригиналу ("but otherwise does nothing except get in the way of enemy fire"), но сути эта фраза не меняет, так что да - её можно просто отрезать.

Reply:
Главное, чтобы информация была донесена до читателя. Близость к оригиналу вторична.